decide/src/main/java/org/example/Decide.java

```diff
@@ -106,7 +106,9 @@ public boolean condition10() {
     }
 
     public boolean condition11() {
-        return true;
+        return settings.NUMPOINTS >= 3
```

Comment:
I feel like this is very similar to condition 5 but maybe it's hard to combine them?

Reply:
Maybe it's not possible without sending arguments to the method since condition 11 has intervening points